monai/networks/blocks/squeeze_and_excitation.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch.nn as nn
+
+from monai.networks.layers.factories import Pool
+
+SUPPORTED_ACTI_1 = {"relu": nn.ReLU, "relu6": nn.ReLU6, "leakyrelu": nn.LeakyReLU}
+
+SUPPORTED_ACTI_2 = {
+    "relu": nn.ReLU,
+    "prelu": nn.PReLU,
+    "relu6": nn.ReLU6,
```

Comment:
I suggest adding `ReLU6` and `Sigmoid`, `Softmax` to our `Act factory`, @ericspod what do you think?
Thanks.

Reply:
Yes we can add the following to `monai/networks/layers/factories.py` and include a few other activation functions provided by Pytorch:

```python
Act.add_factory_callable("relu6", lambda: nn.modules.ReLU6)
Act.add_factory_callable("selu", lambda: nn.modules.SELU)
Act.add_factory_callable("celu", lambda: nn.modules.CELU)
Act.add_factory_callable("gelu", lambda: nn.modules.GELU)
Act.add_factory_callable("sigmoid", lambda: nn.modules.Sigmoid)
Act.add_factory_callable("tanh", lambda: nn.modules.Tanh)
Act.add_factory_callable("softmax", lambda: nn.modules.SoftMax)
Act.add_factory_callable("logsoftmax", lambda: nn.modules.LogSoftMax)
```

The arguments for choosing activation would then be members of `Act`, eg. `acti_type_1: str = Act.RELU`. We could add a check in the constructor to enforce the constraint of which activations to use but I'd leave it open if people wanted to try activations not mentioned in the paper.